recipe/meta.yaml

```diff
@@ -57,8 +62,15 @@ requirements:
     - cuda-version {{ major_version }}.{{ minor_version }}
   run:
     - python
+    # cuda-python requires cuda-nvrtc from the same major version.
+    - cuda-nvrtc >={{ major_version }},<{{ major_version+1 }}.0a0
```

Comment:
Can we defined this version constraint as a variable at the top since we'll use it for both cuda-nvrtc and cuda-cudart?

Reply:
Bradley was having issues getting this working in Jinja, and we may move to `pin_compatible` in a follow-up so we're punting on this change for now.

Reply:
Following up in PR ( https://github.com/conda-forge/cuda-python-feedstock/pull/68 )